joint_limits:
    joint1:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 3000
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: true
      max_effort: 7000
      min_effort: -7000

